crates/primitives/src/proofs.rs

```diff
@@ -100,7 +100,7 @@ pub fn calculate_receipt_root(
 
         return ordered_trie_root_with_encoder(receipts.as_slice(), |r, buf| {
             r.encode_inner(buf, false)
-        })
+        });
```

Comment:
are we doing this again 😭😭😭😭😭😭😭😭😭😭😭

Reply:
bro i legit don't know why it does this. will fmt on linux before merging